portal/src/graphql/portal/SubscriptionScreen.tsx

```diff
@@ -159,7 +159,6 @@ interface SubscriptionPlanCardRenderProps {
   nextBillingDate?: Date;
 }
 
-// eslint-disable-next-line complexity
```

Comment:
Is the complexity rule disabled? Or the algorithm got changed?

Reply:
The complexity rule is not disabled, as you can see there are some new eslint-disable-next-line complexity. Checking the changelog https://github.com/eslint/eslint/blob/main/CHANGELOG.md only gives me this https://github.com/eslint/eslint/commit/2d174532ae96bcaecf6fd7de78755164378b3a2d but I did not update variant in @oursky/eslint-plugin so it should be unrelated. So I have no idea.